02.wo_Defects/ptr_subtraction.c

```diff
@@ -17,7 +17,7 @@
 void ptr_subtraction_001 ()
 {
     int buf1[10];
-	int buf2[5];
+	int *buf2 = buf1 + 5;
```

Comment:
these two buffers are still technically separate objects even though they are most likely adjacent on the stack, so computing pointer difference is still undefined.